ui/models.py

```diff
@@ -88,6 +88,16 @@ def transcode_key(self, preset=None):
         basename, _ = os.path.splitext(original_s3_key)
         return output_template.format(prefix=TRANSCODE_PREFIX, s3key=basename, preset=preset)
 
+    def update_status(self, status):
+        """
+        Assign and save the status of a Video
+
+        Args:
+            status(str): The status to assign the video
+        """
+        self.status = status
+        self.save()
```

Comment:
we should probably open an issue to modify the status field as a list of options